components/automate-ui/src/app/app-routing.module.ts

```diff
@@ -73,7 +73,7 @@ const routes: Routes = [
           component: SettingsLandingComponent
         },
         {
-          path: 'node-lifecycle',
+          path: 'data-lifecycle',
```

Comment:
This change in nomenclature warrants changes to node-lifecycle.md and/or data-lifecycle.md.

Reply:
@kagarmoe / @mjingle - Making sure I don't step on any toes here, is this part of documentation or is this a change that I should handle?

Reply:
I think this is being taken care of already? We're removing any instance of node-lifecycle in the docs per  https://github.com/chef/automate/pull/3170/ 
Also, I'm pretty sure that we're deleting node-lifecycle.md and creating data-lifecycle.md in its place.

Reply:
data lifecycle docs page already existed, but yah deleting the node lifecycle page.. with a redirect.

Reply:
ok great - thats the info I was looking for.  Thank you!